### DISCUSSION TITLE
How to connect JKBMS via bluetooth with password

### DISCUSSION THREAD

**Discussion by paulsteigel** [syssi/esphome-jk-bms#401] [General]
Hi Syssi,
Please guide me on a situation that the BMS Bluetooth is password protected, no longer 1234 anymore?
Thank you very much for your advice.
Ngoc

**Comment by syssi** [comment]
AFAIK the password isn't verified. You can communicate and control the BMS without any password. The android app validates the password on the client side.

**Reply by paulsteigel** [reply]
Thanks Sebastian for your reply! It's clear for me by now.
Ngoc
